docs/xdebug.md

```diff
@@ -43,37 +48,31 @@ This document describes how to enable Xdebug for local PHP development using the
 
     ![Debug](../imgs/xdebug/edit-configurations.png)    
```

Comment:
Image and line 44 above says `docker`, but changed to `Archipelago` below. These should be changed as well to be consistent.

Reply:
Done.